tests/integration/helm_test.go

```diff
@@ -7,6 +7,8 @@ import (
 
 	"github.com/coreos/pkg/capnslog"
 	"github.com/rook/rook/tests/framework/clients"
+	//	"github.com/rook/rook/tests/framework/contracts"
```

Comment:
if this import isn't needed, it should be deleted as opposed to commented out.

Reply:
oops,removing it.